src/lists.h

```diff
@@ -862,17 +863,18 @@ extern UInt SetFiltListTNums [ LAST_REAL_TNUM ] [ LAST_FN + 1 ];
 **
 *F  SET_FILT_LIST( <list>, <fnum> ) . . . . . . . . . . . . . .  set a filter
 */
-#define SET_FILT_LIST(list,fn) \
-  do { \
-    UInt     n; \
-    n = SetFiltListTNums[TNUM_OBJ(list)][fn]; \
-    if ( n != (UInt)-1 ) \
-      RetypeBagIfWritable( list, n ); \
-     else { \
-      Pr( "#E  SET_FILT_LIST[%s][%d] in ", (Int)TNAM_OBJ(list), fn ); \
-      Pr( "%s:%d\n", (Int)__FILE__, (Int)__LINE__); \
-      }  \
-  } while (0)
+static inline void SET_FILT_LIST(Obj list, FilterNumber fn)
+{
+    UInt n = SetFiltListTNums[TNUM_OBJ(list)][fn];
+    if (n == 0) {
+        return;
+    }
```

Comment:
The commit message still says "Add Assertions to *_FILT_LIST macros" and should be rewritten